.chronus/changes/FixXmlLibBug-2024-10-5-17-13-19.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: fix
```

Comment:
```suggestion
changeKind: internal
```

Reply:
as this is a change within the release its not relevant to document that to end users